src/scripts/Update.ts

```diff
@@ -800,6 +800,16 @@ class Update implements Saveable {
 
         '0.9.8': ({ playerData, saveData }) => {
             saveData.oakItemLoadouts = saveData.oakItemLoadouts.map((list, index) => ({ name: `Loadout ${index + 1}`, loadout: list }));
+            //UNTESTED POKERUS UPDATE FUNCTION - MAKE SURE THIS WORKS!!!!
```

Comment:
This is not a comment i want to merge :)
You can bump the version by searching for '0.9.7' and replace it in all non-doc locations (should be 3). Be aware that it will not trigger at build itself, so you need to rebuild manually, or change something else.
And i don't think it works. The value is still "true/false" on my save, after the update has been applied.

Reply:
It probably doesn't work because we replace the property names other than id with a number in the save data, to save space.
Looks like pokerus should be accessed with `p[8]`. probably shouldn't bother using the enum from PartyPokemon in case it changes in the future.

Reply:
Done